homeassistant/components/advantage_air/climate.py

```diff
@@ -53,7 +54,7 @@
 FAN_SPEEDS = {FAN_LOW: 30, FAN_MEDIUM: 60, FAN_HIGH: 100}
 
 ADVANTAGE_AIR_SERVICE_SET_MYZONE = "set_myzone"
-ZONE_HVAC_MODES = [HVAC_MODE_OFF, HVAC_MODE_FAN_ONLY]
+ZONE_HVAC_MODES = [HVAC_MODE_OFF, HVAC_MODE_HEAT_COOL]
```

Comment:
Isn't this a breaking change? Users that call the `climate.set_hvac_mode` service with the fan only mode will need to update their automations or scripts.